docs/guides/wifi-bluetooth.md

```diff
@@ -25,46 +27,158 @@ We now use a script which can help you set up Wi-Fi and Bluetooth. Follow the in
 1. Click [here](../tools/firmware.sh) to download the script.
 2. Boot into macOS.
 3. Run this script there.
-4. When the script shall run successfully, it shall ask you to follow either of the two options mentioned in the [On Linux](#on-linux) section, on Linux.
-5. Boot into Linux.
+
+When you run the script in macOS, it will ask you to choose between 3 methods to move firmware to Linux:
+
+=== "Method 1"
+    #### Method 1: Run the same script on Linux:
```

Comment:
```suggestion
    **Method 1: Run the same script on Linux**
```

Maybe use bold fonts instead of headings so they don't show up in the sidebar while not being clickable. Same goes for the rest.